lib/split/alternative.rb

```diff
@@ -36,31 +40,55 @@ def set_p_winner(prob, goal = nil)
       Split.redis.hset(key, field, prob.to_f)
     end
 
-    def participant_count
-      Split.redis.hget(key, 'participant_count').to_i
+    def participant_count(version_key = version)
+      Split.redis.hget(key(version_key), 'participant_count').to_i
+    end
+
+    def participant_counts
+      (0..version).map do |i|
```

Comment:
Are versions constrained to always start at zero?  Is it possible for a user to manually start at version 1?  If so does this still work as expected?

Reply:
Alternatively, what if the versions go like this  [0, 1, 3]?

Reply:
versions should only ever 1) start from 0) increment by one when reset. This should't occur.

Reply:
Ah, sorry. I was on the impression that a user would specify a version in the experiment definition. This makes sense.